tests/kafkatest/benchmarks/streams/streams_simple_benchmark_test.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.mark import ignore
+
+from kafkatest.tests.kafka_test import KafkaTest
+from kafkatest.services.performance.streams_performance import StreamsSimpleBenchmarkService
+import time
+
+class StreamsSimpleBenchmarkTest(KafkaTest):
+    """
+    Simple benchmark of Kafka Streams.
+    """
+
+    def __init__(self, test_context):
+        super(StreamsSimpleBenchmarkTest, self).__init__(test_context, num_zk=1, num_brokers=1)
+
+        self.driver = StreamsSimpleBenchmarkService(test_context, self.kafka)
+
+    def test_simple_benchmark(self):
+        """
+        Run simple Kafka Streams benchmark
+        """
+
+        self.driver.start()
+        self.driver.wait()
+        self.driver.stop()
+        node = self.driver.node
+        node.account.ssh("grep Performance %s" % self.driver.STDOUT_FILE, allow_fail=False)
+
+        # Collect the data and return it to the framework
+        output = node.account.ssh_capture("grep Performance %s" % self.driver.STDOUT_FILE)
```

Comment:
Instead of putting parsing logic in the test, maybe add it as a method of  `StreamsSimpleBenchmarkService` itself